CODE_SNIPPETS.md

```diff
@@ -0,0 +1,728 @@
+# Code Snippets
+
+## Address Api
+
+### Create
+```bash
+curl https://api.lob.com/v1/addresses \
+  -u test_0dc8d51e0acffcb1880e0f19c79b2f5b0cc: \
+  -d "description=Harry - Office" \
+  -d "name=Harry Zhang" \
+  -d "company=Lob" \
+  -d "email=harry@lob.com" \
+  -d "phone=5555555555" \
+  -d "address_line1=210 King St" \
+  -d "address_line2=# 6100" \
+  -d "address_city=San Francisco" \
+  -d "address_state=CA" \
+  -d "address_zip=94107" \
+  -d "address_country=US"
+```
+
+```java
+LobResponse<Address> response = new Address.RequestBuilder()
+        .setDescription("Harry - Office")
+        .setName("Harry Zhang")
+        .setCompany("Lob")
+        .setLine1("210 King St")
+        .setLine2("# 6100")
+        .setCity("San Francisco")
+        .setState("CA")
+        .setZip("94107")
+        .setCountry("US")
+        .setPhone("555-555-5555")
+        .setEmail("harry@lob.com")
+        .create();
+
+Address address = response.getResponseBody();
+```
+
+### Retrieve
+```bash
+curl https://api.lob.com/v1/addresses/adr_fa85158b26c3eb7c \
+  -u test_0dc8d51e0acffcb1880e0f19c79b2f5b0cc:
+```
+
+```java
+LobResponse<Address> response = Address.retrieve("adr_fa85158b26c3eb7c");
+Address address = response.getResponseBody();
+```
+
+### List
+```bash
+curl -X GET "https://api.lob.com/v1/addresses/?limit=2" \
+  -u test_0dc8d51e0acffcb1880e0f19c79b2f5b0cc:
+```
+
+```java
+Map<String, Object> params = new HashMap<>();
+params.put("limit", 2);
+
+LobResponse<AddressCollection> response = Address.list(params);
+AddressCollection addresses = response.getResponseBody();
+```
+
+### Delete
+```bash
+curl -X DELETE https://api.lob.com/v1/addresses/adr_43769b47aed248c2 \
+  -u test_0dc8d51e0acffcb1880e0f19c79b2f5b0cc:
+```
+
+```java
+$lob->addresses()->delete('adr_43769b47aed248c2');
```

Comment:
Random PHP?

Reply:
Lol, There's alot of code so I'm not surprised I swapped one